Sources/RichTextKit/RichTextView_AppKit.swift

```diff
@@ -209,4 +211,21 @@ public extension RichTextView {
         textStorage
     }
 }
+
+// MARK: - Content Size
+
+extension NSTextView {
```

Comment:
Do we have some solution for UIKit too?

Reply:
Unfortunately not at the moment because I only use it for macos. I'm trying to find a suitable solution for uikit.